### ISSUE TITLE
Example from readme doesn't compile

### ISSUE THREAD

**Issue by janczer** [mgattozzi/github-rs#105]
Hi, I'm very new in rust, and I would like to contribute to this library, but have problem with compile simple example. When I try compile I'm getting an error:

```
error[E0282]: type annotations needed
  --> src/main.rs:14:25
   |
7  |     let me = client.get()
   |         -- consider giving `me` a type
...
14 |             if let Some(json) = json{
   |                         ^^^^ cannot infer type for `_`
```

Thank you

**Comment by janczer** [comment]
Ok, I run the example.

I added this rows:
```
extern crate serde_json;
use serde_json::Value;
```

And change `.execute()` to `.execute::<Value>()`

**Comment by mgattozzi** [comment]
@janczer yes that's what you need to do in order for it to work. Would you like to update the Readme as a first issue with this? It'd be a great first issue and would help others looking to use the library correctly :)

**Comment by janczer** [comment]
Yes, I do it. I should update example/getting_started.rs too?

In order to this issue I think that in this examples:
https://github.com/mgattozzi/github-rs/blob/e07a2124a0b902b03a596bc21826ca46a84a4bff/examples/wrapping_responses.rs#L41

Should be `Ok(result)` ?

**Comment by mgattozzi** [comment]
You should just need to modify README.md file to add the `extern serde_json;` and other lines. You don't need to modify the `github-rs/examples/wrapping_responses.rs` file

**Comment by mgattozzi** [comment]
Resolved with #106! Thanks again @janczer
